earthly/mdlint/Earthfile

```diff
@@ -84,4 +84,4 @@ mdlint-test:
 
     ARG src=$(echo ${PWD}/../../)
 
-    DO +MDLINT_LOCALLY --src=${src}
+    RUN +MDLINT_LOCALLY --src=${src}
```

Comment:
This is a function so use `DO` as it is mentioned in https://docs.earthly.dev/docs/guides/functions  -> "function can be invoked from a target via DO"

Reply:
Roger!

Reply:
thank you for the explanation krub and changed back to DO